modules/installation-gcp-config-yaml.adoc

```diff
@@ -181,6 +182,12 @@ endif::openshift-origin[]
 endif::restricted[]
 ----
 <1> Required. The installation program prompts you for this value.
+<N> Optional: Add this parameter to force the Cloud Credential Operator (CCO) to use the specified mode, instead of having the CCO dynamically try to determine the capabilities of the credentials. For details about CCO modes, see _About the Cloud Credential Operator_ in the _Authentication and authorization_ content.
```

Comment:
This is an <N> because I want to make sure it is correct before shuffling all the callout numbers.

Reply:
+1 this is correct, would it help to specify that CCO is using the root credentials in the kube-system namespace for determining the capabilities? I know this is explained in more detail in the "determining mode" sections so maybe overkill. This should only be relevant to Passthrough and Mint modes when no credentialsMode is specified because CCO will never assume Manual mode on its own afaik.

Reply:
Yeah, I don't think that hurts to mention.